pkgs/development/lua-modules/overrides.nix

```diff
@@ -349,9 +349,8 @@ in
 
     luarocksConfig = lib.recursiveUpdate oa.luarocksConfig {
       variables = {
-        # Can't just be /include and /lib, unfortunately needs the trailing 'mysql'
-        MYSQL_INCDIR = "${libmysqlclient.dev}/include/mysql";
-        MYSQL_LIBDIR = "${libmysqlclient}/lib/mysql";
+        MYSQL_INCDIR = "${libmysqlclient.dev}/include/";
+        MYSQL_LIBDIR = "${libmysqlclient}/lib/";
```

Comment:
@teto looks like this was changed in this bump, it couldn't find what it needed until i removed the trailing `mysql`

Reply:
Maybe we should be using `lib.getDev` and `lib.getLib` here.

Reply:
both suggestions look good

Reply:
Sounds good, updated.